src/validate.py

```diff
@@ -299,6 +306,12 @@ def loop(validation_args_file: str, task_id: str = None):
     last_successful_request_time = [time.time()] * len(task_id_list)
 
     while True:
+        if CLEAN_MODELS_CACHE:
+            for root, dirs, _ in os.walk(file_utils.default_cache_path):
```

Comment:
create a helper function for this to make it clean

Reply:
done
use `click.option` add `--auto_clean_cache` and move clean cache code to a function `clean_model_cache`